components/Buttons/RegularButton.js

```diff
@@ -1,8 +1,20 @@
 import styled, { css } from "styled-components";
 
-export default function RegularButton({ children, onClick, type, variant }) {
+export default function RegularButton({
+  children,
+  onClick,
+  type,
+  variant,
+  disabled,
+}) {
   return (
-    <StyledRegularButton type={type} onClick={onClick} $variant={variant}>
+    <StyledRegularButton
+      type={type}
+      onClick={onClick}
+      $variant={variant}
+      disabled={disabled}
+      $disabled={disabled}
```

Comment:
Hm, I don't completely get how to apply the variant styles then that we have in the ```const variantRegularButtonStyles``` inside the button component. When I put the StyledRegularButton in the styledComponents file and also the const variantRegularButtonStyles for the different stylings of the variants it doesn't apply. All buttons are standard grey then and color is standard black. 

But when we have time for another Refactor US, I will have another look into that. Due to time issues I will check that later more thoroughly.

Reply:
Thats fine, it likely is another problem like missing props when you use the Button (you need to pas $variant instead of variant for example). For now its good enough.